style.css

```diff
@@ -0,0 +1,93 @@
+body {
+background: url("http://wp.givingtuesday.ca/app/uploads/2014/11/GivingTuesday_BlogHeader.jpg") no-repeat center center fixed;
+background-size: cover;
+background-color: #F4F4FC;
+color: #14146A;
+font-family: 'Open Sans', Arial, Helvetica, sans-serif;
+font-size: 32px;
+line-height: 1.5em;
+}
+
+
+a { 
+	text-decoration: none; 
+}
+h1 { 
+	font-size: 1em; 
+}
+
+h1, p {
+margin-bottom: 10px;
+}
+strong {
+font-weight: bold;
+}
+.uppercase { text-transform: uppercase; }
+
+/* ---------- LOGIN ---------- */
+#login {
+margin: 100px auto;
+width: 300px;
+}
+form fieldset input[type="text"], input[type="password"] {
+background-color: #e5e5e5;
+border: none;
+border-radius: 3px;
+-moz-border-radius: 3px;
+-webkit-border-radius: 3px;
+color: #5a5656;
+font-family: 'Open Sans', Arial, Helvetica, sans-serif;
+font-size: 14px;
+height: 50px;
+outline: none;
+padding: 0px 10px;
+width: 280px;
+-webkit-appearance:none;
+}
+form fieldset input[type="submit"] {
+background-color: #008dde;
+border: none;
+border-radius: 3px;
+-moz-border-radius: 3px;
+-webkit-border-radius: 3px;
+color: #f4f4f4;
+cursor: pointer;
+font-family: 'Open Sans', Arial, Helvetica, sans-serif;
+height: 50px;
+text-transform: uppercase;
+width: 300px;
+-webkit-appearance:none;
+}
+form fieldset a {
+color: #5a5656;
+font-size: 10px;
+}
+form fieldset a:hover { text-decoration: underline; }
+.btn-round {
+background-color: #5a5656;
+border-radius: 50%;
+-moz-border-radius: 50%;
+-webkit-border-radius: 50%;
+color: #f4f4f4;
+display: block;
+font-size: 12px;
+height: 50px;
+line-height: 50px;
+margin: 50px 150px;
+text-align: center;
+text-transform: uppercase;
+width: 50px;
+}
+
+.SignUp {
```

Comment:
The naming convention of classes is not consistent. In some places you have used lower case for class names, whereas here you have used camel case convention. Stick to one.